js/index.js

```diff
@@ -38,5 +40,94 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+//nav selector
+const [ nav ] = document.getElementsByTagName('nav');
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// CTA Selector
+const [ cta ] = document.getElementsByClassName('cta');
+const [ ctaH1 ] = document.getElementsByTagName('h1');
+const [ ctaBtn ] = document.getElementsByTagName('button');
+const ctaImg = document.getElementById('cta-img');
+
+//Main Content Selector
+const [ mc ] = document.getElementsByClassName('main-content');
+
+//Middle Image Selector
+const middleImg = document.getElementById('middle-img');
+
+//Bottom Content Selector
+const [ bc ] = document.getElementsByClassName("bottom-content");
+
+// Text Content H4 and P Content Selectors
+const textContentH4 = document.querySelectorAll(".text-content h4");
+const textContentP = document.querySelectorAll(".text-content p");
+
+// Contact H4 and P Content Selectors
+const contactH4 = document.querySelectorAll(".contact h4");
+const contactP = document.querySelectorAll(".contact p");
+
+// Footer
+const footerP = document.querySelectorAll("footer p");
+
+// Mutate the Dom
+
+//Header
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+var newLink1 = document.createElement("a");
+newLink1.textContent = siteContent["nav"]["nav-item-7"];
+
+var newLink2 = document.createElement("a");
+newLink2.textContent = siteContent["nav"]["nav-item-8"];
+
+nav.appendChild(newLink1);
+nav.prepend(newLink2);
+
+Array.from(nav.children).forEach((value, index) => {
+  value.textContent = siteContent["nav"][`nav-item-${index + 1}`];
+  value.setAttribute('style', "color:green;");
```

Comment:
You could also use the `style` object on the node element and set CSS properties directly. Like so:
```js
value.style.color = 'green';
```